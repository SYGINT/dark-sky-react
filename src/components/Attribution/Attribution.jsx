import React from "react";

import "./Attribution.scss";

function Attribution() {
  return (
    <div className="attributions">
      <a href="https://darksky.net/poweredby/" alt="Powered by Dark Sky">
        Powered by Dark Sky
      </a> with <a
        href="https://www.amcharts.com/free-animated-svg-weather-icons/"
        alt="Icons by amCharts"
      >
        Icons
      </a>{" "}
      by{" "}
      <a href="http://amcharts.com" alt="amCharts website">
        amCharts
      </a>
    </div>
  );
}

export default Attribution;
