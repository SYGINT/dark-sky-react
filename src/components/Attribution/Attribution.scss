.attributions {
  text-align: center;
  margin-top: 30px;
  line-height: 5rem;

  span:after {
    margin: 0 10px;
    display: inline-block;
    content: "â˜‚";
    color: #000;
  }
}
