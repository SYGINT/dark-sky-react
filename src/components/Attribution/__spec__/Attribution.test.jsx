import React from "react";
import { shallow } from "enzyme";

import Attribution from "../";

it("renders without crashing", () => {
  const wrapper = shallow(<Attribution />);
  expect(wrapper).toMatchSnapshot();
});
