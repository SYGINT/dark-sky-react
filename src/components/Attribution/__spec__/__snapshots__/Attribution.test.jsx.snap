// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="attributions"
>
  <span
    className="attribution"
  >
    <a
      alt="Powered by Dark Sky"
      href="https://darksky.net/poweredby/"
    >
      Powered by Dark Sky
    </a>
  </span>
  <a
    alt="Icons by amCharts"
    href="https://www.amcharts.com/free-animated-svg-weather-icons/"
  >
    Icons
  </a>
   
  by
   
  <a
    alt="amCharts website"
    href="http://amcharts.com"
  >
    amCharts
  </a>
</div>
`;
